src/pages/how-it-works.astro

```diff
@@ -4,25 +4,141 @@ import Layout from "../layouts/Layout.astro";
+                    
+                    </ul>
+                    
+                            <img src="/assets/images/hpp-third.svg" alt="" class="hpp__image">
+                            <p class="hpp__image-disclaimer">Image created for Alexion Pharmaceuticals, Inc. for illustrative purposes.</p>
+                  </div>
+              
+              
+                <div class="hpp__block">
+
+                <ul class="hpp__list">
+                  <li class="hpp__item">ExcExcess PLP results in deficient vitamin B6 in the CNS, which can cause neurologic symptoms like seizures (in infants), fatigue, and mood disorders</li>
+                  <li class="hpp__item-nosub modifier">Phosphoethanolamine (PEA) levels are also elevated in patients with hypophosphatasia. To date, the effects of PEA are not fully understood</li>
+                    
+                    </ul>
+                    
+                            <img src="/assets/images/hpp-fourth.svg" alt="" class="hpp__image">
+                  
+                  </div>
+                </div>
+  </Section>
+
+  <Section classnames="moa">
+    
+      <p id="STRENSIQ-MOA" class="custom-section__title" slot="header">STRENSIQ mechanism of action (MOA)</p>
+      <div>
+
+        <h1 class="hpp__title">Only STRENSIQ replaces the deficient alkaline phosphatase enzyme</h1>
+        <h2 class="hpp__sub-title">Replacing alkaline phosphatase enables bone mineralization and reduces harmful levels of substrates.</h2>
+
+        <div class="hpp__container">
+        <div class="hpp__block">
+
+          <ul class="hpp__list">
+            <li class="hpp__item-nosub">STRENSIQ cleaves Pi from PPi so Pi can combine with calcium to form hydroxyapatite crystal, the foundation of strong bone</li>
+              
+              </ul>
+              
+                      <img src="/assets/images/hpp-fifth.svg" alt="" class="hpp__image modifier">
+                      <p class="hpp__image-disclaimer">Image created for Alexion Pharmaceuticals, Inc. for illustrative purposes.</p>
+            
+            </div>
+ 
+
+            <div class="hpp__block">
+
+              <ul class="hpp__list">
+                <li class="hpp__item">STRENSIQ reduces elevated levels of systemic PPi and PLP (the active form of vitamin B6)</li>
+                  <li class="hpp__sub-item modifier">Reductions in plasma PPi and PLP did not correlate with clinical outcomes</li>
+                  <li class="hpp__sub-item modifier">The effect of STRENSIQ on PEA was not measured in clinical trials</li>
+                  </ul>
+                  
+                          <img src="/assets/images/hpp-sisxth.svg" alt="" class="hpp__image">
+                       
+                </div>
+              </div>
+
+
+
+              <div class="without-strensiq">
+                <h1 class="without-strensiq__title">Without STRENSIQ</h1>
```

Comment:
Change this to an h2, also check the spacing elements are too close to each other.

![image](https://user-images.githubusercontent.com/6466646/232895492-fecc1397-5dd2-45a9-aafa-d5fb9dcaf2d7.png)